zoo-hope/src/components/navBar/index.tsx

```diff
@@ -4,20 +4,26 @@ import "../../styles/index.scss";
 import { useState } from "react";
 import LogoNavBar from "../../images/logo/logo.png"
 import { elements } from "../../data/nav";
+import { useTranslation } from "react-i18next";
+import "../../i18n/i18n"
 
 export const NavBar = () => {
-
+  const { t, i18n } = useTranslation();
   const [clickedButtonUA, setClickedButtonUA] = useState(true);
```

Comment:
Please rework, to have only 1 state for language
It is better to have enum {
UA,
EN
}

Reply:
2 states are about 1 item, so there is no need to have 2